File: app/src/main/java/com/platdmit/simplecloudmanager/data/database/dao/LoadAverageDao.java
package com.platdmit.simplecloudmanager.data.database.dao;

import com.platdmit.simplecloudmanager.data.database.entity.DbLoadAverage;

import java.util.List;

import androidx.room.Dao;
import androidx.room.Query;

@Dao
public interface LoadAverageDao extends BaseDao<DbLoadAverage>{
    @Query("SELECT * FROM dbloadaverage WHERE mServer = :serverId")
    List<DbLoadAverage> getLoadAverageForServer(long serverId);
}

File: data/src/main/java/com/platdmit/data/database/dao/LoadAverageDao.kt
package com.platdmit.data.database.dao

import androidx.room.Dao
import androidx.room.Query
import com.platdmit.data.database.entity.DbLoadAverage

@Dao
interface LoadAverageDao : BaseDao<DbLoadAverage> {
    @Query("SELECT * FROM dbloadaverage WHERE server = :serverId")
    fun getLoadAverageForServer(serverId: Long): List<DbLoadAverage>?
}